src/datasource_toolkit/forestadmin/datasource_toolkit/decorators/computed/utils/flattener.py

```diff
@@ -65,7 +65,7 @@ def unflatten(flats: FlatRecordList, projection: List[str]) -> List[RecordsDataA
     for record_index in range(num_records):
         for path_index, path in enumerate(projection):
             # When a marker is found, the parent is null.
-            parts = [*filter(lambda part: part != _MARKER_NAME, path.split(":"))]
+            parts = [*filter(lambda part: part not in (_MARKER_NAME, "*"), path.split(":"))]
```

Comment:
There is no modification in flattener in ruby. Is this useless ??

Reply:
Yes 👍